pyomo/solvers/plugins/solvers/GUROBI_RUN.py

```diff
@@ -258,25 +258,39 @@ def gurobi_run(model_file, warmstart_file, soln_file, mipgap, options, suffixes)
         solnfile.write('objective: %s\n' % str(obj_value))
         solnfile.write('gap: 0.0\n')
 
-        for var in vars:
-            solnfile.write('var: %s : %s\n' % (str(var.getAttr(GRB.Attr.VarName)), str(var.getAttr(GRB.Attr.X))))
+        vals = model.getAttr("X", vars)
+        names = model.getAttr("VarName", vars)
+        for val, name in zip(vals, names):
```

Comment:
I recommend adding `from six.moves import zip` for better Py2 performance

Reply:
@ghackebeil Good suggestion. Thanks. I will update this.